scripts/setup-operators.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash
+set -x
+echo "Starting sealed secrets operator installation"
+install_sealed_secrets_operator(){
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  generateName: cicd-
+  namespace: cicd
+spec:
+  targetNamespaces:
+  - cicd
+EOF
+
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sealed-secrets-operator-helm
+  namespace: cicd
+spec:
+  channel: alpha
+  name: sealed-secrets-operator-helm
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+EOF
+}
+
+install_sealed_secrets_operator
+# sealed secrets operator status check
+count=0
+while [ "$count" -lt "5" ];
+do
+    operator_status=`oc get csv -n cicd | grep sealed-secrets-operator`
+    if [[ $operator_status == *"Succeeded"* ]]; then
+        break
+    else
+        count=`expr $count + 1`
+        sleep 10
+    fi
+done
+echo "Completes sealed secrets operator installation"
+
+echo "Starting sealed secrets operator instance creation"
+create_sealed_secrets_operator_instance(){
+oc create -f - <<EOF
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecretController
+metadata: 
+  name: sealedsecretsontroller
+  namespace: cicd
+spec: 
+  affinity: {}
+  controller: 
+    create: true
+  crd: 
+    create: true
+    keep: true
+  image: 
+    pullPolicy: IfNotPresent
+    repository: "quay.io/bitnami/sealed-secrets-controller@sha256:8e9a37bb2e1a6f3a8bee949e3af0e9dab0d7dca618f1a63048dc541b5d554985"
+  ingress: 
+    annotations: {}
+    enabled: false
+    hosts: 
+      - chart-example.local
+    path: /v1/cert.pem
+    tls: []
+  networkPolicy: false
+  nodeSelector: {}
+  podAnnotations: {}
+  podLabels: {}
+  priorityClassName: ""
+  rbac: 
+    create: true
+    pspEnabled: false
+  resources: {}
+  secretName: sealed-secrets-key
+  securityContext: 
+    fsGroup: ""
+    runAsUser: ""
+  serviceAccount: 
+    create: true
```

Comment:
There look like a lot of default fields in this map, can we reduce this?

Reply:
Will remove and validate it. Thanks